dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -514,6 +515,23 @@ def test_correct_null_row_counts(self):
         self.assertEqual(3, profile.row_is_null_count)
         self.assertEqual(3/24, profile._get_row_is_null_ratio())
 
+
+    def test_null_in_file(self):
+        filename_null_in_file = os.path.join(
+            test_root_path, 'data','csv/stripe-payments.csv')
+        profiler_options = ProfilerOptions()
+        profiler_options.set({'data_labeler.is_enabled': False})
+        data = dp.Data(filename_null_in_file)
+        profile = dp.Profiler(data, profiler_options=profiler_options)
+
+        report = profile.report(report_options={"output_format":"pretty"})
+        
+        self.assertEqual(
+            report['data_stats']['invoice_id']['statistics']['null_types_index'],
+            {'': "['']"}
```

Comment:
I am not familiar with null_types_index, but it says in the readme it maps a string to a list of ints, but this maps a string to a string (of a list of strings). So I feel like either this test case or the readme may be incorrect.

Reply:
It seems the error is only with the report not making it serializable, should this instead go with all the report tests? Maybe it doesn't matter.

Reply:
It does seem weird that the final value is an empty string for the list of values.

Reply:
I believe the value should be: "[53]"